guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -82,8 +82,8 @@ Complete the following steps on the connected {ProjectServer}.
 ----
 {RepoRHEL7ServerAnsible}
```

Comment:
In this document you're mixing RHEL 7 & 8 which probably won't work.

Reply:
@ekohl is it ok to remove the RHEL 7 repos that aren't related to 6.12?

Reply:
Well, here I don't see any EL8 instructions so the whole section would be empty, right? The entire chapter needs to be updated.

Reply:
We don't need a dedicated Ansible on EL8 anymore, so this can just go  (and the later mention of it), and then the section is fine again?